src/core/server.js

```diff
@@ -800,7 +810,10 @@ Server.prototype._sendMessage = function(message) {
     if (this._remote.trace) {
       log.info(this.getServerID(), 'request:', message);
     }
-    this._ws.send(JSON.stringify(message));
+    this._ws.send(JSON.stringify(message), () => {
+      // sometimes gitves 'not opened'
```

Comment:
I'm not sure, but it seems like we should probably either resend or throw an error. I think it's usually dangerous to let things fail silently.